README.md

```diff
@@ -1,9 +1,8 @@
 <a name="readme-top"></a>
 <div align="center">
-  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
   <br/>
 
-  <h3><b>Island Blogs</b></h3>
+  <h3><b>Island Blog</b></h3>
```

Comment:
- [ ] [OPTIONAL] I would like to strongly recommend you customize your license link by adding your name and your project title in order to make it more professional 👍